donkeycar/management/makemovie.py

```diff
@@ -236,11 +240,10 @@ def make_frame(self, t):
         a frame counter. This assumes sequential access.
         '''
 
-        if self.iRec >= self.end or self.iRec >= len(self.index):
+        if self.current >= self.end_index:
             return None
 
-        rec_ix = self.index[self.iRec]
-        rec = self.tub.get_record(rec_ix)
+        rec = self.iterator.next()
         image = rec['cam/image_array']
```

Comment:
This needs to load the image as `rec['cam/image_array']` only returns the path:
`
        rec = self.iterator.next()
        img_path = os.path.join(self.tub.images_base_path, rec['cam/image_array'])
        image = img_to_arr(Image.open(img_path))
`

Reply:
I will move it to the `Tub` eventually.